wurst/systems/entities/bosses/AlligatorMan.wurst

```diff
@@ -26,6 +44,40 @@ public class AlligatorMan extends Hostile
     override function getNumCorpses() returns int
         return 10
 
-
 init
     AlligatorMan.initialize()
+
+    // Cycle through alligator regions
+    for currentRegion in regions
+        // Create trigger for detecting alligator
+        let x =CreateTrigger()
+        x.registerEnterRegion(currentRegion, null)
```

Comment:
Don't assign this trigger, just use `..` notation.

    CreateTrigger()
        ..registerEnterRegion(...)
        ..addAction() ->
            // some code here